shell/components/SortableTable/index.vue

```diff
@@ -345,27 +345,7 @@ export default {
       this.searchQuery = q;
     }, 200),
 
-    descending(neu, old) {
-      this.watcherUpdateLiveAndDelayed(neu, old);
-    },
-
-    searchQuery(neu, old) {
-      this.watcherUpdateLiveAndDelayed(neu, old);
-    },
-
-    sortFields(neu, old) {
-      this.watcherUpdateLiveAndDelayed(neu, old);
-    },
-
-    groupBy(neu, old) {
-      this.watcherUpdateLiveAndDelayed(neu, old);
-    },
-
-    namespaces(neu, old) {
-      this.watcherUpdateLiveAndDelayed(neu, old);
-    },
-
-    page(neu, old) {
+    displayRows(neu, old) {
```

Comment:
@aalves08 We deliberately did not use displayRows, as this updates too frequently.